routes/error-tracker.js

```diff
@@ -49,6 +49,51 @@ function ignoreMessageOrException(message, exception) {
 }
 
 /**
+ * @desc converts stack traces and standardizes them to chrome like.
+ * @param {string} exception
+ * @return {string} standardized exception
+ */
+function stackTraceConversion(exception) {
+  let chromeStackTraceRegex = /^\s*at (?:([^]*) )?([^]+):(\d+):(\d+)/m;
+  let match = chromeStackTraceRegex.test(exception);
+  if (match) {
+    exception = exception.substring(exception.indexOf('\n'));
+    let exceptions = exception.split('\n');
+    let validExceptions = exceptions.filter(function(value) {
+      return chromeStackTraceRegex.test(value);
+    });
+    exception = validExceptions.join('\n');
+    return exception;
+  } else if (!match) {
+    let mozillaSafariStackTraceRegex = /^([^@]*)@(.+):(\d+):(\d+)$/m;
```

Comment:
Instead of splitting and filtering, we can just design this regex to match compatible stack frames, and iterate with it.